analysis/floodscan/01_floodscan.md

```diff
@@ -1,25 +1,202 @@
 # Floodscan
 
+Looking at floodscan for Yemen generall, and
+comparing to CCCM
+
 ```python
 %load_ext jupyter_black
 ```
 
 ```python
-from src import utils
+from datetime import timedelta
+
+import pandas as pd
+import matplotlib.pyplot as plt
```

Comment:
`matplotlib` should be added into `requirements.txt`?

Reply:
good catch, fixed